### PR TITLE
Can work with Paperclip gem now

### PR DESCRIPTION
**Pull request by dwa012**
Not sure if this is being actively maintained or not.

I made some small changes so this provider will work with the Rails Paperclip gem.

I did try to run the tests, but the were some issues with them starting with the VCR stuff.

Cheers,
Daniel

### CODE REVIEW

**Comment by plribeiro3000** [comment]
@dwa012 Could you fix the indentation issues at your changes?

Thanks!

**Comment by dwa012** [comment]
No problem. I made my edits in vim which caused the indentation to be all
screwy. Will get it updated today.

**Comment by dwa012** [comment]
@plribeiro3000 The indentation should be good now

File: lib/fog/dropbox/models/storage/files.rb [hunk 1]
```diff
@@ -32,6 +32,7 @@ def head(key, options = {})
 
         def new(attributes = {})
           requires :directory
+          attributes['path'] ||= attributes[:key]
```
Comment (plribeiro3000) [inline lib/fog/dropbox/models/storage/files.rb:35]: Wouldn't be better if you use a symbol key and maintain consistency?

Comment (plribeiro3000) [inline lib/fog/dropbox/models/storage/files.rb:35]: And what does that mean? Why are you assigning the value of `:key` to `:path`?

Comment (dwa012) [inline lib/fog/dropbox/models/storage/files.rb:35]: That was an attempt to get the correct path from the attributes passed via Paperclip.
Paperclip has a fog adapter implementation which passes `:key` for the file path of the file. Not sure if that is non standard or not. 
They work with a number of providers already.

By assigning it to `'path'`,  I was avoiding breaking existing code that may pass in that key as a string.

From the other provider implementations, `:key` seems to be correct. Just didn't want to cause any breaking changes for others.

**Comment by plribeiro3000** [comment]
Sounds good to me!

**Comment by plribeiro3000** [comment]
@zshannon Could you do a new release please?

Thanks!

**Pull request merged**